borg/archiver.py

```diff
@@ -56,6 +56,9 @@ def print_verbose(self, msg, *args, **kw):
         msg = args and msg % args or msg
         logger.info(msg)
 
+    def print_status(self, status, path):
+        logger.debug("%1s %s", status, remove_surrogates(path))
```

Comment:
wasn't it previous behaviour that one -v did output all files it considered? so I suggest you keep that until we have a better means to control output (topical flags).